webrender/src/prim_store.rs

```diff
@@ -718,10 +718,12 @@ impl PrimitiveStore {
 
     pub fn resolve_primitives(&mut self,
                               resource_cache: &ResourceCache,
-                              device_pixel_ratio: f32) -> Vec<DeferredResolve> {
+                              device_pixel_ratio: f32,
+                              layer_store: &Vec<StackingContext>,
+                              auxiliary_lists_map: &AuxiliaryListsMap) -> Vec<DeferredResolve> {
```

Comment:
Yeah, I didn't really like the aux list thing either. But we did a bunch of stuff after cull_layers and cull_layers was already decently big. We can refactor later?